lib/inline_svg.rb

```diff
@@ -81,6 +88,11 @@ def incompatible_transformation?(klass)
       !klass.is_a?(Class) || !klass.respond_to?(:create_with_value) || !klass.instance_methods.include?(:transform)
     end
 
+    def set_asset_finder_from_callable
+      while @asset_finder&.respond_to?(:call)
```

Comment:
What's the significance of the `while` here? 🤔

Reply:
To handle a callable returning a callable. We can replace it with a conditional to take a single call.

Reply:
I added an example.